# Overall successful exposure calibration
int8_t SUCCESS = 1

# Failure to call service '/set_parameters'
int8_t CAMERA_EXPOSURE_SET_FAILURE = -1

# Failure to receive Image from callback
int8_t IMAGE_NOT_RECEIVED_FAILURE = -2

# Requested camera exposure is out of bounds failure
int8_t REQUESTED_CAMERA_EXPOSURE_OUT_OF_BOUNDS_FAILURE = -3

# Requested camera blue value is out of bounds failure
int8_t REQUESTED_CAMERA_BLUE_VALUE_OUT_OF_BOUNDS_FAILURE = -4

# Requested camera blue value hasn't been achieved yet
int8_t REQUESTED_CAMERA_BLUE_VALUE_NOT_MET = -5
